pkg/types/openstack/platform.go

```diff
@@ -17,14 +17,17 @@ type Platform struct {
 	Cloud string `json:"cloud"`
 
 	// ExternalNetwork is name of the external network in your OpenStack cluster.
-	ExternalNetwork string `json:"externalNetwork"`
+
+	// TODO: we also have to touch pkg/asset/store/data/install.openshift.io_installconfigs.yaml\
+	// TODO: need to update documentation at docs/user/openstack/customization.md
+	ExternalNetwork string `json:"externalNetwork,omitempty"`
```

Comment:
remove

Reply:
setting it to omitempty is right

Reply:
I think it should be `omitempty` to allow empty value